packages/components/pages/f-mfa/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-mfa",
   "description": "Fozzie Mfa - Multi-factor Authenticator Input Form",
-  "version": "0.2.0",
+  "version": "0.1.1",
```

Comment:
This version is lower than the previous version

Reply:
Should be `0.3.0`